app/assets/stylesheets/main.scss

```diff
@@ -7,3 +7,10 @@
 
 // Lastly import partials
 @import "partials/chessboard";
+
+.gravatar {
+  border-radius: 50%;
+  -webkit-border-redius: 50;
+  -moz-border-redius: 50;
+  margin-top: 5px;
+}
```

Comment:
Can we move this out of `main.scss`, which is just supposed to be a manifest and into its own file in `partials/gravatars.scss` - we can then simply include this line here:

`@import partials/gravatar` - if we can make this simple change, it will be more consistent with the way we are trying to organize the CSS.

Otherwise looks fantastic! :+1: